browser-interface/packages/shared/meta/sagas.ts

```diff
@@ -37,6 +42,38 @@ export function* metaSaga(): any {
   }
 
   yield put(metaConfigurationInitialized(merge))
+
+  yield fork(fetchInitialPortableExperiences)
+}
+
+function* fetchInitialPortableExperiences() {
```

Comment:
This PR reverts an important fix for race conditions needed for the fashion week. Please do not include the reverted commit of https://github.com/decentraland/unity-renderer/pull/4611 in `dev`

Reply:
We need to make a fix properly then as it does not work on Standalone according to QA

Reply:
So what's the plan @menduz @eordano ? We can do it via `dev` and then we can align `main` with `dev` or make a `hotfix`?